dataland-frontend/src/components/resources/newLandingPage/TheIntro.vue

```diff
@@ -133,6 +186,32 @@ const aboutIntroSection = computed(() => {
         line-height: 32px;
       }
     }
+    .back-button {
+      position: absolute;
+      top: -44px;
+      left: 32px;
+      cursor: pointer;
+      z-index: 10;
+      text-align: left;
+      padding: 0 16px;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 600;
+      line-height: 24px;
+      letter-spacing: 0.25px;
+      &::before {
+        content: "";
+        display: block;
+        position: absolute;
+        left: -8px;
+        top: 50%;
+        transform: translateY(-50%);
+        width: 16px;
+        height: 16px;
+        background-image: url(/static/icons/Arrow--right.svg);
+        transform: rotateY(180deg) translateY(-50%);
+      }
+    }
```

Comment:
can we recycle the same button from the AuthSection here or add this styling to the global scss button styles ?

Reply:
Yes, that would be ideal but in this ticket I have no more time. Please open a new ticket with all these optimisations.

Reply:
okay